### PR TITLE
Jointable

### PR DESCRIPTION
**Pull request by kennyegun24**
In this PR, I...
- Created new join tables
- I added data to the tables
- I wrote some querries

image

image

image

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,102 @@
...
+
+SELECT MAX(escape_attempts) FROM animals
+GROUP BY neutered;
+
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals
+GROUP BY species;
+
+SELECT AVG(escape_attempts) FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+
```
Comment (1GORDON) [inline queries.sql:58]: Great job @kennyegun24 🤝

- [ ] Kindly add all your queries to this `queries.sql` file.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @kennyegun24 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] You may also use the correct readme file. Kindly use this [readme template](https://github.com/microverseinc/readme-template) as specified in the project requirements and don't forget to customize it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kennyegun24** [comment]
@1GORDON, Good day to you...
This project is a database project, so we were required to use the database-type readme template and not the normal readme template

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @kennyegun24 , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice query 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**